asdf/tags/core/tests/test_ndarray.py

```diff
@@ -96,14 +96,16 @@ def test_byteorder(tmpdir):
         }
 
     def check_asdf(asdf):
-        tree = asdf.tree
+        my_tree = asdf.tree
+        for endian in ('bigendian', 'little'):
+            assert my_tree[endian].dtype == tree[endian].dtype
```

Comment:
I assume this is unrelated but just for a bit more coverage?

Reply:
The old code shadowed `tree` from the outer scope, also that test failed when I debugged the test suite because the dtype-s did not match completely, so I spent some fun time here.